Source/WTF/wtf/PlatformWPE.cmake

```diff
@@ -28,6 +28,26 @@ list(APPEND WTF_SOURCES
     unix/UniStdExtrasUnix.cpp
 )
 
+list(APPEND WTF_PUBLIC_HEADERS
+    glib/ChassisType.h
+    glib/GMutexLocker.h
+    glib/GRefPtr.h
+    glib/GSocketMonitor.h
+    glib/GThreadSafeWeakPtr.h
+    glib/GTypedefs.h
+    glib/GUniquePtr.h
+    glib/GWeakPtr.h
+    glib/RunLoopSourcePriority.h
+    glib/Sandbox.h
+    glib/SocketConnection.h
+    glib/WTFGType.h
+
+    linux/CurrentProcessMemoryStatus.h
+    linux/ProcessMemoryFootprint.h
+
+    unix/UnixFileDescriptor.h
+)
```

Comment:
It's a bit of a pity that we have this duplicated for GTK and WPE. It would be nice to have this in a `GLib.cmake` file, but alas that can be done in a follow-up.